src/vellum/workflows/references/output.py

```diff
@@ -78,7 +78,7 @@ def _as_generator(self, node_output: Queue) -> Generator[_OutputType, None, Type
     def __eq__(self, other: object) -> bool:
         if not isinstance(other, type(self)):
             return False
-        return super().__eq__(other) and self._outputs_class == other._outputs_class
+        return super().__eq__(other) and id(self._outputs_class) == id(other._outputs_class)
```

Comment:
Our Outputs class has custom equality method that we deferred editing and deferred investigating the root bug there, in case there were dependent consumers on how that equality evaluated today